pkg/api/v1/atlasproject_types.go

```diff
@@ -54,12 +55,14 @@ type AtlasProjectSpec struct {
 
 // AtlasProjectStatus defines the observed state of AtlasProject
 type AtlasProjectStatus struct {
-	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	status.Common `json:",inline"`
```

Comment:
[nit] `,inline` is not "officially" supported in json tags and there are still discussions going around it

Reply:
this is a requirement of one of the code generating libs we use - it requires all the elements to have a `json` annotation

Reply:
Oh, interesting then nothing to do here